docs/api/v2/node.md

```diff
@@ -74,6 +74,16 @@ proc info*(node: WakuNode): WakuInfo =
   ##
   ## Status: Implemented.
   ##
+
+proc resume*(node: WakuNode, peerList: Option[seq[PeerInfo]]) =
```

Comment:
I find the resume name not very descriptive. In the case of store protocol it might sense (kind of?) but at node level it doesn't.

Maybe `resumeFetchHistory`? or something that describes the intent better.

Reply:
Maybe `resume` is fine actually, since all other 7 API calls are simple verbs https://github.com/status-im/nim-waku/blob/master/docs/api/v2/node.md don't know what others think?